File: articles/algorithms-in-systems-modeling/c/stock_flow_update.c
#include <stdio.h>

int main(void) {
    double current_stock = 100.0;
    double inflow = 12.0;
    double outflow = 7.0;
    double next_stock = current_stock + inflow - outflow;
    printf("next_stock=%.4f\n", next_stock);
    return 0;
}

File: articles/algorithms-in-systems-modeling/rust/stock_flow_update.rs
fn main() {
    let current_stock = 100.0_f64;
    let inflow = 12.0_f64;
    let outflow = 7.0_f64;
    let next_stock = current_stock + inflow - outflow;
    println!("next_stock={:.4}", next_stock);
}
